test/package/Vagrantfile

```diff
@@ -0,0 +1,27 @@
+Vagrant.configure("2") do |config|
+  config.vm.synced_folder '../../target/', '/packages'
+
+  config.vm.define "deb" do |deb|
+    deb.vm.box = 'elastic/debian-9-x86_64'
+    deb.vm.provision "ansible" do |ansible|
+      ansible.playbook = "deb.yml"
+    end
+    deb.vm.network "private_network", ip: "192.168.50.5"
+  end
+
+  config.vm.define "rpm" do |rpm|
+    rpm.vm.box = 'elastic/centos-7-x86_64'
```

Comment:
We might want to start with centos 8, since it's current.

Reply:
I have a request to get updated virtual machines, we're a little out of date with availability.

https://app.vagrantup.com/elastic

Reply:
Ah, ok - then we can proceed with this.